client/api_bucket.go

```diff
@@ -761,3 +777,54 @@ func (c *client) CancelMigrateBucket(ctx context.Context, bucketName string, opt
 
 	return c.SubmitProposal(ctx, []sdk.Msg{cancelBucketMsg}, opts.ProposalDepositAmount, opts.ProposalTitle, opts.ProposalSummary, types.SubmitProposalOptions{Metadata: opts.ProposalMetaData, TxOption: opts.TxOpts})
 }
+
+// ListBucketsByPaymentAccount list bucket by payment account
+func (c *client) ListBucketsByPaymentAccount(ctx context.Context, paymentAccount string, opts types.ListBucketsByPaymentAccountOptions) (types.ListBucketsByPaymentAccountResult, error) {
+
```

Comment:
check if it is a valid paymentAccount
for example :

Reply:
principal, err := sdk.AccAddressFromHexUnsafe(principalAddr)
	if err != nil {
		return "", err
	}

Reply:
fixed